### PR TITLE
Badges and tests

### PR DESCRIPTION
**Pull request by ccremer**
As discussed in internal ticket VT-1069, here are some badges and some sample unit tests

### CODE REVIEW

**Comment by srueg** [comment]
I don't have merge rights on this repo.

**Comment by ccremer** [comment]
I have reworked the Docker build setup and removed multiarch support for Docker. Care to review again?

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # Image Cleanup Client
 
+![](https://img.shields.io/docker/pulls/appuio/image-cleanup)
+![](https://img.shields.io/github/v/release/appuio/image-cleanup)
+![](https://img.shields.io/github/license/appuio/image-cleanup)
+
```
Comment (bittner) [inline README.md]: I'd suggest to align that with how we did it on some of the other container repositories, e.g.

* [container-oc](https://github.com/appuio/container-oc/blob/master/README.md)
* [docker-concierge](https://github.com/vshn/docker-concierge/blob/master/README.md)
* [docker-modulesync](https://github.com/vshn/docker-modulesync/blob/master/README.md)
* [modsecurity-docker](https://github.com/vshn/modsecurity-docker/blob/master/README.md)

**Comment by ccremer** [comment]
wait...pipeline will fail for master

**Comment by ccremer** [comment]
As discussed with @srueg , we will not do docker pushes on master branch anymore, only tags. As such, I have adapted the pipeline so that docker builds & pushes also happens with goreleaser

**Pull request merged**